src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class FIgureSupplier {
```

Comment:
The class definition should not start with an empty line according to the checklist.